Jenkinsfile

```diff
@@ -302,6 +304,11 @@ pipeline {
                                 exit $c
                                 '''
                             }
+                            post {
+                                always {
+                                    junit testResults: 'bundles/**/*.xml', allowEmptyResults: true
```

Comment:
I'm concerned with how the junit plugin processes xml files. It [warns](https://wiki.jenkins.io/display/JENKINS/JUnit+Plugin):
> Be sure not to include any non-report files into this pattern.

And we definitely have xml files that are not report files under the bundles directory:
```
bundles/test-integration/1/TestAuthZPluginV2BadManifestFailsDaemonStart/d26edc53d5db9/root/plugins/20f707871fe12c665ed3685fde0f870ec5d15e1b9d7ee4d154f166ab91944131/rootfs/usr/local/go/doc/codewalk/functions.xml
bundles/test-integration/1/TestAuthZPluginV2BadManifestFailsDaemonStart/d26edc53d5db9/root/plugins/20f707871fe12c665ed3685fde0f870ec5d15e1b9d7ee4d154f166ab91944131/rootfs/usr/local/go/doc/codewalk/markov.xml
bundles/test-integration/1/TestAuthZPluginV2BadManifestFailsDaemonStart/d26edc53d5db9/root/plugins/20f707871fe12c665ed3685fde0f870ec5d15e1b9d7ee4d154f166ab91944131/rootfs/usr/local/go/doc/codewalk/codewalk.xml
bundles/test-integration/1/TestAuthZPluginV2BadManifestFailsDaemonStart/d26edc53d5db9/root/plugins/20f707871fe12c665ed3685fde0f870ec5d15e1b9d7ee4d154f166ab91944131/rootfs/usr/local/go/doc/codewalk/sharemem.xml
```

I suggest chaging the glob to something like `bundles/**/*-junit-report.xml`

Reply:
Good catch! Yes, `.xml` files could be present at this point, because container/plugin rootfs's may still be around.

I'll update that matching